docs/releases/4.0.txt

```diff
@@ -58,6 +58,13 @@ For example::
 Functional unique constraints are added to models using the
 :attr:`Meta.constraints <django.db.models.Options.constraints>` option.
 
+``scrypt`` password hasher
+--------------------------
+
+The new :ref:`scrypt password hasher <scrypt-usage>` is more secure and
+recommended over PBKDF2. However, it's not the default as it requires the
+OpenSSL 1.1+ and much memory which can cause issues.
```

Comment:
'requires the OpenSSl' -> 'requires OpenSSl'

'much memory' -> 'much more memory' or maybe just 'more memory'.

Reply:
and a comma before "which"

Reply:
> Or maybe just finish at memory. It's already implied from the however there are "issues".

Yes. 

Or maybe just finish at memory. I think it's implied from the however there are 'issues' with what follows.